### ISSUE TITLE
Accessibility Audit: Navigator Onboarding

### ISSUE THREAD

**Issue by carye3** [newjersey/navigator.business.nj.gov#4569]
**Problem Statement:** Perform an audit on the Navigator onboarding flows to document and understand how the feature compares to current A compliance.

**Benefit Hypothesis:** With comms doing a marketing push on Navigator soon, it would be beneficial to fix any accessibility issues. - "Digital access needs are sometimes treated as a feature, but we're not leaving out a feature, we're leaving out people."

**Success Criteria:** First audit of the onboarding flow has been started. Bonus if it is completed

**Out-of-scope:** Analysis and sharing of findings

**Assumptions:** 
- We are checking for A compliance (per discussion on 10/24)
- We will have some dev support to perform the audit if needed
- We have access to the tools we need to perform the audit. Example, [screen readers ](https://assistivlabs.com/pricing)
- Accessibility readout [document](https://docs.google.com/document/d/16M25bJlfoDjjwPW_fl7O8EeOzdOslGEB2k8PPtLprfA/edit#) 

**Feature Readiness Check List:**
- [x] Document the scope of the audit by working with devs (Faraz) to outline the current onboarding flows
- [x] Define which standards are applicable to the product (do this on [WCAG 2.2 audit doc](https://docs.google.com/spreadsheets/d/1E3bKKG4vI1ucWkUW4HNTMdy9wOeLz9Ti/edit?usp=sharing&ouid=116116843699891540443&rtpof=true&sd=true))
- [x] (bonus) See how we could utilize the Azure tool
- [ ] Conduct audit of items that the design team can test - [checklist here](https://docs.google.com/spreadsheets/d/1ivZ_JulqB_Am7B4YdPQTkOXddlB4QigD/edit#gid=1254264225)
- [x] Provide an update on Accessibility Plan to PMs/stakeholders

**Comment by amandabeall** [comment]
Notes from 11/1 Dev/Design sync

**[Flow in Figma](https://www.figma.com/file/Qs8qQrlVdl3ajcWpwcvnGP/44---50.-Navigator-Designs?node-id=20264%3A178071)**
- Decided to determine the priority of personas to test and to start there. Priority is defined as:
    - Starting a new business (Poppy)
    - Exiting NJ based business (Oscar)
    - Out of State business (Dakota)
- The devs noted that many components are shared in all three flows. There was a discussion about testing all the flows fully vs just testing the remaining components. It was decided that the best way to test will be by flow 

**[WCAG checklist](https://docs.google.com/spreadsheets/d/1ivZ_JulqB_Am7B4YdPQTkOXddlB4QigD/edit#gid=1254264225)**
- Validated with the devs that the criteria labeled as "not applicable" were not applicable to Onboarding 
- Due to time and knowledge, this audit will be limited to the design team and what they can test independently (ex- we cannot check Aria labels because we don't have screen readers). We determined this was okay because devs already use automatic tools that check _most_ of these items. 
- Dillion offered to set up a space in Azure to better document findings between all the flows. Hoping to have something by end of week

**Comment by amandabeall** [comment]
Jess provided data for users on the site by persona. Priority list above has been adjusted to match 
<img width="884" alt="Screen Shot 2022-11-01 at 4 24 54 PM" src="https://user-images.githubusercontent.com/114253061/199337098-4ba18ef5-f8e6-4165-ba11-fb2d54cca0ed.png">

**Comment by amandabeall** [comment]
We have started the audit! 

We will be testing out the Azure tool to document our findings. All the tests are under the [Design Audit Task](https://dev.azure.com/dillonsullivan/Product%20Portfolio/_workitems/edit/105) and results will be reported out once complete

**Comment by amandabeall** [comment]
We did not complete the audit due to other priorities. We have created a new ticket for us to continue this work - https://dev.azure.com/dillonsullivan/Product%20Portfolio/_workitems/edit/180

This ticket is resolved
